MODULE.bazel

```diff
@@ -5,11 +5,26 @@ module(
     repo_name = "rules_kotlin",
 )
 
-bazel_dep(name = "platforms", version = "0.0.6")
-bazel_dep(name = "bazel_skylib", version = "1.4.2")
-bazel_dep(name = "rules_java", version = "7.2.0")
-bazel_dep(name = "rules_python", version = "0.23.1")
-bazel_dep(name = "rules_cc", version = "0.0.8")
+bazel_dep(
```

Comment:
Can you undo the formatting changes in this file?

Reply:
pre-commit adds this. is there a way i can bypass this?